braze/client.py

```diff
@@ -151,9 +151,9 @@ def user_delete(self, external_ids):
 
     def user_export(self, external_ids=None, email=None, fields_to_export=None):
         """
-        Export user profiles from braze. One of ``external_ids`` or ``email`` must be
-        provided. Braze allows exporting multiple user profiles through ``external_ids``
-        but only one with the ``email`` argument.
+        Export user profiles from braze. One or both of ``external_ids`` or ``email``
+        must be provided. Braze allows exporting multiple user profiles through
+        ``external_ids`` but only one with the ``email`` argument.
```

Comment:
Q.
> Braze allows exporting multiple user profiles through ``external_ids`` but only one with the ``email`` argument.

If only one user can be exported if email is provided then wondering what's the reason behind specifying both external_ids and email. I suppose as an extra layer of confirmation (set intersection)  i.e. "Only export a user if Braze finds a user for this email and for one of these external ids, otherwise don't"?

Reply:
Yeah don't know what use case they were thinking about. But it's useful for us. Think of this case:
We get an email in the coupon email view and we have profile that is currently in use.
By calling export with the `grx_profile_id` and the `email` we can figure out if:
- Email belongs to this profile
- Email belongs to a different profile
- Email has not been seen before
Depending on the case we want to:
- Just email the current profile
- Email the profile it belongs to
- Create a new profile, assign it this email and then email the new profile.